app/lib/server/methods/addOAuthService.js

```diff
@@ -29,7 +29,7 @@ Meteor.methods({
 		settings.add(`Accounts_OAuth_Custom-${ name }-identity_path`          , '/me'             , { type: 'string' , group: 'OAuth', section: `Custom OAuth: ${ name }`, i18nLabel: 'Accounts_OAuth_Custom_Identity_Path', persistent: true });
 		settings.add(`Accounts_OAuth_Custom-${ name }-authorize_path`         , '/oauth/authorize', { type: 'string' , group: 'OAuth', section: `Custom OAuth: ${ name }`, i18nLabel: 'Accounts_OAuth_Custom_Authorize_Path', persistent: true });
 		settings.add(`Accounts_OAuth_Custom-${ name }-scope`, 'openid', { type: 'string', group: 'OAuth', section: `Custom OAuth: ${ name }`, i18nLabel: 'Accounts_OAuth_Custom_Scope', persistent: true });
-		settings.add(`Accounts_OAuth_Custom-${ name }-param_name`, 'access_token', { type: 'string', group: 'OAuth', section: `Custom OAuth: ${ name }`, i18nLabel: 'Accounts_OAuth_Custom_Param_Name', persistent: true });
+		settings.add(`Accounts_OAuth_Custom-${ name }-access_token_param`, 'access_token', { type: 'string', group: 'OAuth', section: `Custom OAuth: ${ name }`, i18nLabel: 'Accounts_OAuth_Custom_Access_Token_Param', persistent: true });
```

Comment:
can you align these with the comma's above?

Reply:
ok